server.js

```diff
@@ -1,30 +1,80 @@
-import express from "express";
-import cors from "cors";
+import express from 'express'
+import cors from 'cors'
+import bodyParser from 'body-parser'
+import listEndpoints from 'express-list-endpoints'
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
+import netflixDatas from './data/netflix-titles.json'
+// console.log(netflixDatas)
 
-// Defines the port the app will run on. Defaults to 8080, but can be overridden
+// Defines the port the app will run on. Defaults to 8080, but can be overrtitleden
 // when starting the server. Example command to overwrite PORT env variable value:
 // PORT=9000 npm start
-const port = process.env.PORT || 8080;
-const app = express();
+const port = process.env.PORT || 8080
+const app = express()
 
-// Add middlewares to enable cors and json body parsing
-app.use(cors());
-app.use(express.json());
+//parseInt or number or + will converrt string to numbers
+
+// Add mtitledlewares to enable cors and json body parsing
+app.use(cors())
+app.use(express.json())
+app.use(bodyParser.json())
 
 // Start defining your routes here
-app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
-});
+app.get('/', (req, res) => {
+  res.send(listEndpoints(app))
+})
+// Geting my first request
+app.get('/myNetflix', (req, res) => {
+  res.status(200).json({
+    data: netflixDatas,
+    success: true,
+  })
+})
+// Filter on movies or tv shows
+app.get('/myNetflix/type/:type', (req, res) => {
+  const showTypes = netflixDatas.filter(
+    (data) => data.type.toLowerCase() === req.params.type.toLowerCase(),
```

Comment:
Works to type either "Movie" or "TV show", although it is only returning one single element. Since type is a very generic value, I think it would be convenient to return a collection of results (all movies or all TV-show).